pkgs/development/python-modules/sqlalchemy-views/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+  # required deps
+, setuptools
+, sqlalchemy
+  # test deps
+, pytestCheckHook
+, pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "sqlalchemy-views";
+  version = "0.3.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "jklukas";
+    rev = "5f959ed4a898068fd6073a452c5a13f4e19ea59e";
+    hash = "sha256-MJgikWXo3lpMsSYbb5sOSOTbJPOx5gEghW1V9jKvHKU=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+  propagatedBuildInputs = [ sqlalchemy ];
+  nativeCheckInputs = [ pytestCheckHook pytest-cov ];
+
+  pythonImportsCheck = [ "sqlalchemy_views" ];
```

Comment:
- Expand list items to one per line
- Patch out calls to pytest-cov and drop it from `nativeCheckInputs`

Reply:
> Patch out calls to pytest-cov and drop it from nativeCheckInputs

This seems like unnecessary additional work. Any particular reason we can't leave it as is?

Reply:
We generally don't test coverage, since it is only meaningful for upstream.

Reply:
I guess it's not too bad. Pushing up the changes.

Reply:
https://github.com/jklukas/sqlalchemy-views/blob/main/tox.ini#L26

The `--cov` arguments here.

Reply:
Done.